sql/base/realmd.sql

```diff
@@ -1,8 +1,8 @@
--- MySQL dump 10.13
+-- MySQL dump 10.13  Distrib 5.6.17, for Win64 (x86_64)
 --
--- Host: localhost    Database: realmd
+-- Host: 127.0.0.1    Database: realmd3
 -- ------------------------------------------------------
--- Server version	5.5.32
+-- Server version 5.6.21
```

Comment:
These details don't really matter. As said elsewhere, autogenerated file. I'll change the database name at least before it's final. ;)